browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -182,14 +206,25 @@ void BraveToolbarView::Update(content::WebContents* tab) {
     bookmark_->SetVisible(browser_defaults::bookmarks_enabled &&
                           edit_bookmarks_enabled_.GetValue());
   }
+  if (speedreader_) {
+    speedreader_->SetVisible(true);
```

Comment:
View is visible by default.
I think this button's visibility is determined by `SpeedreaderService::IsEnabled()`.

Reply:
just removed that line, if it exists it is visible